fdbserver/WorkerInterface.actor.h

```diff
@@ -416,26 +416,38 @@ struct GetWorkersRequest {
 	}
 };
 
+struct InitializeTLogTeam {
```

Comment:
Follow the convention and name this as `InitializeTLogTeamRequest`

Reply:
Changed it to TLogGroup.